manage_breast_screening/templates/record_a_mammogram/appointment_cannot_go_ahead.jinja

```diff
@@ -5,6 +5,7 @@
 
 {% set title = "Appointment cannot go ahead" %}
 {% set caption = participant.full_name %}
+{% set decision_legend = "Does the appointment need to be rescheduled?" %}
```

Comment:
I think you'll need to rebase this, as part of the presenter refactor I moved all these variables into the view